src/pki/est/est_tls.c

```diff
@@ -272,6 +279,22 @@ METHOD(est_tls_t, request, bool,
 			}
 		}
 
+		if (content_b64enc) {
+			char *filtered = malloc(data.len);
+
+			int cnt = 0;
+			for (int i = 0; i < data.len; i++) {
+				char c = data.ptr[i];
+				if (c != '\r' && c != '\n' &&
+					c != '\t' && c != ' ') {
+					filtered[cnt] = data.ptr[i];
+					cnt++;
+				}
+			}
+
+			data = chunk_create(filtered, cnt);
```

Comment:
This could be simplified (`i` has also to be delared where `len` is):

```suggestion
			for (i = 0, len = 0; i < data.len; i++)
			{
				if (!isspace(data.ptr[i]))
				{
					data.ptr[len++] = data.ptr[i];
				}
			}
			data.len = len;
```

Reply:
Thank you for the feedback and pointer to the errata and previous removal.

The topic came up as base64 encoded response containig CRLF could not be parsed/validated successfully.

I'll update the PR within the next week.

Reply:
I've updated the PR to include the suggested change and have rebased it on the current `master`.

The proposed solution to re-use/modify the existing `chunk` is elegant and simple. :+1: